examples/http3-client.c

```diff
@@ -244,6 +253,16 @@ static void recv_cb(EV_P_ ev_io *w, int revents) {
                 break;
             }
 
+            if (!settings_received) {
+                int rc = quiche_h3_for_each_setting(conn_io->http3,
+                                                    for_each_setting,
+                                                    NULL);
+
+                if (rc == 0) {
+                    settings_received = true;
+                }
```

Comment:
since `quiche_h3_for_each_setting()` always return 0, I think there is no need to check the return value.
```suggestion
                quiche_h3_for_each_setting(conn_io->http3,
                                                    for_each_setting,
                                                    NULL);

                settings_received = true;
```